doc/modules/classes.rst

```diff
@@ -161,28 +196,45 @@ Classes
    :toctree: generated/
    :template: class.rst
 
-   cross_validation.KFold
-   cross_validation.LabelKFold
-   cross_validation.LabelShuffleSplit
-   cross_validation.LeaveOneLabelOut
-   cross_validation.LeaveOneOut
-   cross_validation.LeavePLabelOut
-   cross_validation.LeavePOut
-   cross_validation.PredefinedSplit
-   cross_validation.ShuffleSplit
-   cross_validation.StratifiedKFold
-   cross_validation.StratifiedShuffleSplit
+   model_selection.GridSearchCV
+   model_selection.RandomizedSearchCV
 
+Parameter search utilites
+-------------------------
+
+.. currentmodule:: sklearn
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   model_selection.ParameterGrid
+   model_selection.ParameterSampler
+
+Scorers
+-------
+
+.. currentmodule:: sklearn
 
 .. autosummary::
    :toctree: generated/
    :template: function.rst
 
-   cross_validation.train_test_split
-   cross_validation.cross_val_score
-   cross_validation.cross_val_predict
-   cross_validation.permutation_test_score
-   cross_validation.check_cv
+   model_selection.cross_val_score
+   model_selection.cross_val_predict
+   model_selection.permutation_test_score
+   model_selection.check_cv
+
+Diagnostic utilities to evaluate the Bias-Variance trade-off
+------------------------------------------------------------
```

Comment:
As an alternative section title if not merged, I would keep the previous title ` Learning curve evaluation`.